library/core/src/mem/mod.rs

```diff
@@ -678,6 +678,58 @@ pub unsafe fn uninitialized<T>() -> T {
     }
 }
 
+/// Create a fresh instance of the inhabited ZST type `T`.
```

Comment:
```suggestion
/// Create a fresh instance of the inhabited zero-sized type `T`.
```

zero-sized type type 🙃

Reply:
Good point -- we don't need more ATM machines that make you type your PIN number.

And it's better not to repeat an acronym from the name in the short description too.